backend/app/auth/routes.py

```
@@ -0,0 +1,241 @@
+from fastapi import APIRouter, HTTPException, status, Depends
```

Ruff (0.11.9)

`fastapi.Depends` imported but unused

Remove unused import: `fastapi.Depends`

(F401)